### PR TITLE
Use HTML5 version of uploader

### PR DESCRIPTION
**Pull request by orangejenny**
Flash version is breaking staging for reasons beyond my ken.

Tested in Chrome and Firefox, both with & without the changes made in https://github.com/dimagi/commcare-hq/pull/10319.

@sravfeyn / @emord

### CODE REVIEW

**Comment by benrudolph** [comment]
interesting. much better fix than my hack

File: hqmedia.upload_controller.js [hunk 1]
```diff
@@ -13,12 +13,9 @@ function BaseHQMediaUploadController (uploader_name, marker, options) {
 
     ///// YUI Uploader Specific Params
     // Strip hostname from URL in case it's included from CDN: http://stackoverflow.com/a/6945443/835696
```
Comment (emord) [inline hqmedia.upload_controller.js]: This comment can be removed

Comment (orangejenny) [inline hqmedia.upload_controller.js]: fixed

**Comment by emord** [comment]
Nice! I think we also need to update vellum (probably just a rebuild with new mediauploader master) before this is deployed to prod.

**Comment by orangejenny** [comment]
@emord sounds good, i can do vellum-to-master before today's deploy

**Pull request merged**